pkg/setting/setting_unified_alerting.go

```diff
@@ -103,31 +103,39 @@ func (cfg *Cfg) readUnifiedAlertingEnabledSetting(section *ini.Section) (*bool,
 	// At present an invalid value is considered the same as no value. This means that a
 	// spelling mistake in the string "false" could enable unified alerting rather
 	// than disable it. This issue can be found here
-	unifiedAlerting, err := section.Key("enabled").Bool()
-	if err != nil {
+	hasEnabled := section.Key("enabled").Value() != ""
```

Comment:
You need to update this to "capture" now that #50350 has merged.

Reply:
That's for screenshots no?

Reply:
yep, my bad.